src/gtk/textctrl.cpp

```diff
@@ -2153,13 +2191,24 @@ wxSize wxTextCtrl::DoGetSizeFromTextSize(int xlen, int ylen) const
     //multiline
     else
     {
+        // add space for vertical scrollbar
+        if ( m_scrollBar[1] && !(m_windowStyle & wxTE_NO_VSCROLL) )
+            tsize.IncBy(GTKGetPreferredSize(GTK_WIDGET(m_scrollBar[1])).x + 3, 0);
```

Comment:
what is this `3`? Same for horizontal scrollbar.

Reply:
@MaartenBent ,
I have no idea. This is an original code.

Reply:
What do you mean? You added it here. Why?